### PR TITLE
[CI] Add the context in the failed device bots.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tools/devops/automation/templates/common/publish-html.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ steps:
         $targetUrl = $Env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI + "$Env:SYSTEM_TEAMPROJECT/_build/index?buildId=$Env:BUILD_BUILDID&view=ms.vss-test-web.test-result-details"
 
         New-GitHubComment -Header "Tests were not ran ($Env:CONTEXT)." -Emoji ":warning:" -Description "Results were skipped for this run  due to provisioning problems [Azure Devops]($targetUrl). Please contact the bot administrator."
-        Set-GitHubStatus -Status "error" -Description "Provisioning bot for tests failed." -Context "$(Build.DefinitionName) (Test run)"
+        Set-GitHubStatus -Status "error" -Description "Provisioning bot for tests failed." -Context "$Env:CONTEX - $(Build.DefinitionName) (Test run)"
```
Comment (spouliot) [inline tools/devops/automation/templates/common/publish-html.yml:72]: typo ? it's `($Env:CONTEXT)` above

-> 🛌

**Review by spouliot — CHANGES_REQUESTED** [review]

File: tools/devops/automation/templates/common/publish-html.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/common/publish-html.yml:72]: That is because above it a comment and has the context in  brackets :)

Comment (mandel-macaque) [reply tools/devops/automation/templates/common/publish-html.yml:72]: Oh fun.. i have to go to sleep

Comment (mandel-macaque) [inline tools/devops/automation/templates/common/publish-html.yml:72]:
```suggestion
        Set-GitHubStatus -Status "error" -Description "Provisioning bot for tests failed." -Context "$Env: CONTEXT - $(Build.DefinitionName) (Test run)"
```

File: tools/devops/automation/templates/common/publish-html.yml [hunk 2]
```diff
@@ -69,7 +69,7 @@ steps:
         $targetUrl = $Env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI + "$Env:SYSTEM_TEAMPROJECT/_build/index?buildId=$Env:BUILD_BUILDID&view=ms.vss-test-web.test-result-details"
 
         New-GitHubComment -Header "Tests were not ran ($Env:CONTEXT)." -Emoji ":warning:" -Description "Results were skipped for this run  due to provisioning problems [Azure Devops]($targetUrl). Please contact the bot administrator."
-        Set-GitHubStatus -Status "error" -Description "Provisioning bot for tests failed." -Context "$(Build.DefinitionName) (Test run)"
+        Set-GitHubStatus -Status "error" -Description "Provisioning bot for tests failed." -Context "$Env: CONTEXT - $(Build.DefinitionName) (Test run)"
```
Comment (mandel-macaque) [inline tools/devops/automation/templates/common/publish-html.yml:72]:
```suggestion
        Set-GitHubStatus -Status "error" -Description "Provisioning bot for tests failed." -Context "$Env:CONTEXT - $(Build.DefinitionName) (Test run)"
```

**Review by dalexsoto — APPROVED** [review]
All the typos!

**Review by spouliot — APPROVED** [review]

**Pull request merged**